engine/wwtlib/HipsProperties.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace wwtlib
+{
+
+    public class HipsProperties
+    {
+        public Dictionary<string, string> Properties = new Dictionary<string, string>();
+        public VoTable VoTable = null;
+        public bool downloadComplete = false;
+        private WebFile webFile;
+        public static HipsProperties GetPropertiesCat(string url)
```

Comment:
Is this function used at all? I don't think so.

Reply:
I thought I was going to need it for Catalog HiPS, but that turned out to be false. I'll remove it